src/test/java/hw3/shape/Circle.java

```diff
@@ -0,0 +1,17 @@
+package hw3.shape;
```

Comment:
All these classes should be moved to src/main/java/hw3/shape package. and in the test package you can just leave your current Instances class (probably would be worth to rename it to ShapesTest)